packages/@tinacms/cli/src/utils/script-helpers.test.ts

```diff
@@ -0,0 +1,62 @@
+/**
+
+Copyright 2021 Forestry.io Holdings, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+import { generateGqlScript, extendNextScripts } from './script-helpers'
+
+describe('generateGqlScript', () => {
+  it('wraps original script correctly', () => {
+    const newScript = generateGqlScript('yarn dev -p 3000')
+
+    expect(newScript).toEqual('yarn tinacms server:start -c "yarn dev -p 3000"')
```

Comment:
We probably do not want to have "yarn in here" only tinacms server:start -c "what was in there before".

The  `yarn` is not needed

Reply:
Ah good catch, just removed it